hydra-node/test/Hydra/Model/MockChain.hs

```diff
@@ -308,20 +290,20 @@ scriptLedger seedInput =
  where
   initUTxO = fromPairs [(seedInput, (arbitrary >>= genTxOutAdaOnly) `generateWith` 42)]
 
+  -- XXX: We could easily add 'slot' validation here and this would already
+  -- emulate the dropping of outdated transactions from the cardano-node
+  -- mempool.
   applyTransactions !slot utxo = \case
```

Comment:
Was it the case that the slot is not used here? I seem to remember that from one of the sessions.

Reply:
Yest it's not used right now. This is just a note that we could use it here to increase capabilities of the mock chain.